sklearn/cross_decomposition/pls_.py

```diff
@@ -432,27 +432,24 @@ def predict(self, X, copy=True):
         Ypred = np.dot(X, self.coef_)
         return Ypred + self.y_mean_
 
-    def fit_transform(self, X, y=None, **fit_params):
```

Comment:
Yes, I had a discussion with @TomDLT and he told me I could remove it. What do you want to do about it? From what I understand, if a user uses this argument it will be used in fit() and since it's not available in fit, it will provoke an error. So I'm not sure how removing it could break someone's code?